opendebates/admin.py

```diff
@@ -15,9 +17,33 @@ class CategoryAdmin(ModelAdmin):
 @register(models.Submission)
 class SubmissionAdmin(ModelAdmin):
     list_display = [f.name for f in models.Submission._meta.fields]
-    actions = [admin_list_export]
+    list_filter = ('approved', )
+    search_fields = ('idea', )
+    actions = [admin_list_export, 'remove_submissions']
     raw_id_fields = ['voter', 'duplicate_of']
 
+    def remove_submissions(self, request, queryset):
+        "Custom action to mark submissions 'unapproved' and to notify users by email."
+        if request.POST.get('post'):
+            count = 0
+            # only email user if submission changes status, hence the filter
+            for submission in queryset.filter(approved=True):
+                count += 1
+                submission.approved = False
+                submission.moderated_removal = True
+                submission.removal_flags.all().update(reviewed=True)
+                submission.save()
+                send_email("idea_is_removed", {"idea": submission})
+            if count == 1:
+                msg = "Removed 1 submission"
+            else:
+                msg = "Removed {} submissions".format(count)
+            self.message_user(request, msg)
+            return None  # returning None causes us to return to changelist
+        context = {'queryset': queryset}
+        return render(request, 'opendebates/admin/remove_submissions.html', context)
+    remove_submissions.short_description = 'Remove selected submissions'
```

Comment:
Now I'm thinking we should get rid of the built-in delete option.  The second step of the process looks & feels broadly similar in both actions, so they would actually need to attend closely to the text on the second screen to know which one they're being prompted to confirm.  And I'm just worried that with moderators being so overwhelmed, they're going to stop reading that confirmation page closely sooner or later.